<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WgtSettingsS</class>
 <widget class="QWidget" name="WgtSettingsS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WgtSettingsS</string>
  </property>
  <layout class="QGridLayout" name="MainLayout" columnstretch="0,0,0">
   <item row="8" column="2">
    <widget class="QLineEdit" name="SetupPathEdit">
     <property name="placeholderText">
      <string>设置默认安装目录，初始目录: &quot;c:/HZLH/&quot;</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="label_18">
     <property name="text">
      <string>环境变量</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="CompanyEdit">
     <property name="placeholderText">
      <string>公司名</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>项目</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>公司</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>v</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="VersionEdit1"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="VersionEdit2"/>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="VersionEdit3"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="ProjectCB">
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ProjectEdit">
       <property name="placeholderText">
        <string>设置项目名</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="label_17">
     <property name="text">
      <string>写注册表</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>版本</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>快捷方式</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_15">
     <property name="text">
      <string>卸载图标</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>源文件夹</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>语言</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>基础配置</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_16">
     <property name="text">
      <string>HeadBmp</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>安装目录</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <layout class="QHBoxLayout" name="LanguageLayout"/>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>安装图标</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>高级配置</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>输出文件</string>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QLabel" name="label_19">
     <property name="text">
      <string>依赖包</string>
     </property>
    </widget>
   </item>
   <item row="14" column="2">
    <widget class="QCheckBox" name="PathCB">
     <property name="text">
      <string>将安装路径写入环境变量</string>
     </property>
    </widget>
   </item>
   <item row="15" column="2">
    <widget class="QCheckBox" name="DependsCB">
     <property name="text">
      <string>添加依赖包</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
